packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -138,11 +139,35 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
         readOnly: true,
         reflectToAttribute: true,
       },
+
+      /**
+       * The object used to localize this component.
+       * To change the default localization, replace the entire
+       * `i18n` object with a custom one.
+       *
+       * The object has the following structure and default values:
+       * ```
+       * {
+       *   toggle: 'Toggle child items'
+       * }
+       * ```
+       *
+       * @type {SideNavItemI18n}
+       * @default {English/US}
+       */
+      i18n: {
```

Comment:
I would have expected to be able to set this on `vaadin-side-nav` itself, having to set it on each item seems impractical?

Reply:
Good point, thanks. We could have both, as we do in `vaadin-avatar-group`. I will update the PR.